ansible/playbook/tasks/update.yml

```diff
@@ -8,6 +8,7 @@
     pmm_packages:
       - percona-grafana
       - percona-prometheus
+      - percona-VictoriaMetrics
```

Comment:
`percona-victoriametrics`  or  `percona-victoria-metrics` for consistency?

Reply:
Renamed to `percona-victoriametrics`